Translation: Bank transfer (#168)
pretix/pretix-docs: docs/guides/payment/bank-transfer.de.md

﻿# Banküberweisung

Banküberweisungen sind eine der zahlreichen Möglichkeiten, in pretix Zahlungen abzuwickeln.

> Zahlungen per Banküberweisung gehen direkt auf Ihrem Bankkonto ein.

Reviewer: > Zahlungen per Banküberweisung gehen direkt auf Ihrem Bankkonto ein.

Ich halte "direkt" im DE für missverständlich, weil es eine temporale Interpretation zulässt (welche falsch wäre und durch "directly" im EN weniger entsteht). Finden wir ein anderes Wort, welches dieses Problem vermeidet?

Author: ```suggestion
Zahlungen per Banküberweisung gehen ohne Umwege auf Ihrem Bankkonto ein.
```

oder 

```suggestion
Zahlungen per Banküberweisung gehen ohne einen zwischengeschalteten Zahlungsdienstleister auf Ihrem Bankkonto ein.
```

Standardmäßig überwacht pretix nicht, welche Zahlungen auf Ihrem Bankkonto eingehen.

> Wenn Sie pretix Hosted nutzen, können Sie Ihr Bankkonto über unsere Integration mit GoCardless verbinden, damit Bankdaten automatisch importiert werden.

Reviewer: ```suggestion
Wenn Sie pretix Hosted nutzen, können Sie über unsere Integration mit GoCardless Ihr Bankkonto verbinden, damit Bankdaten automatisch importiert werden.
```

Sie haben folgende Alternativen, um pretix über Zahlungseingänge zu informieren: Sie geben Zahlungen manuell als abgeschlossen frei oder Sie importieren regelmäßig digitale Kontoauszüge.
Dieser Artikel erklärt, wie Sie die Verbindung zu einem Bankkonto einrichten, um darauf Zahlungen über pretix zu empfangen.
Sie erfahren außerdem, wie Sie pretix über Zahlungseingänge informieren.

!!! Hinweis

>     pretix führt keine automatischen Erstattungen per Überweisung durch.

Author: ```suggestion
    pretix wickelt Erstattungen per Banküberweisung nicht automatisch ab.
```

>     Wenn ein Kund\*in eine Bestellung storniert, die Erstattung über die zuvor verwendete Zahlungsmethode erhalten möchte und diese Zahlungsmethode eine Banküberweisung war, werden Sie per E-Mail benachrichtigt.

Author: ```suggestion
    Wenn ein\*e Kund\*in eine Bestellung storniert, die Erstattung über die zuvor verwendete Zahlungsmethode anfordert und diese Zahlungsmethode eine Banküberweisung war, dann werden Sie per E-Mail benachrichtigt.
```

    Sie müssen die Erstattung dann manuell vornehmen.

## Voraussetzungen

> Sie richten Zahlungsdienstleister auf Veranstaltungsebene ein. Zuerst müssen Sie also eine Veranstaltung anlegen.

Author: Ein Satz pro Zeile
```suggestion
Sie richten Zahlungsdienstleister auf Veranstaltungsebene ein. 
Zuerst müssen Sie also eine Veranstaltung anlegen.
```

Reviewer: Vielleicht auch im Deutschen eingängig genug, um ein Satz zu sein

Sie benötigen Zugriff auf das Bankkonto, das Sie mit pretix verwenden möchten.

## Anleitung

Folgende Schritte sind erforderlich, um in pretix Banküberweisungen als Zahlungsmethode zu nutzen:

 1. Zum Einrichten von Banküberweisungen die Erweiterung für Banküberweisungen aktivieren und die Einstellungen für die Überweisungen anpassen
 2. Den automatischen Import von Transaktionen einrichten **oder** regelmäßig digitale Kontoauszüge importieren
 3. Transaktionen manuell freigeben
 4. Transaktionen manuell bearbeiten

> Dieser Abschnitt erklärt diese Schritte ausführlich.

Reviewer: > Dieser Abschnitt erklärt diese Schritte ausführlich.

"Abschnitt" ist als Übersetzung für "section" sehr naheliegend, aber in der Praxis ist das gerade schon ein ganzer Artikel.

Author: Das bezieht sich auf den Abschnitt ## Anleitung, daher stimmt das schon. Aber vielleicht würde folgende Änderung es präzisieren: 

```suggestion
Die folgenden Abschnitte erklären diese Schritte ausführlich.
```

### Banküberweisungen einrichten

Wenn Sie in pretix Zahlungen per Banküberweisung erhalten möchten, muss die Erweiterung "Banküberweisung" aktiviert sein.

> Sie prüfen folgendermaßen nach, ob die Erweiterung aktiv ist: Navigieren Sie zu :navpath:Ihre Veranstaltung → :fa3-wrench: Einstellungen → Erweiterungen:.

Reviewer: > Sie prüfen folgendermaßen nach, ob die Erweiterung aktiv ist: Navigieren Sie zu :navpath:Ihre Veranstaltung → :fa3-wrench: Einstellungen → Erweiterungen:.

Fände eine Konstruktion wie im EN („Um zu überprüfen, …“) auch okay

Author: ```suggestion
Um zu prüfen, ob die Erweiterung aktiv ist, navigieren Sie zu :navpath:Ihre Veranstaltung → :fa3-wrench: Einstellungen → Erweiterungen:.

```

> Wechseln Sie zum Reiter :btn:Zahlungsdienstleister:.

Author: UI-Text
```suggestion
Wechseln Sie zum Reiter :btn:Zahlungsmethoden:.
```

Auf dieser Seite wird die Erweiterung "Banküberweisung" oben angezeigt.
Sie sollte standardmäßig aktiv sein.
Wenn sie aktiv ist, hat sie ein grünes Tag ":fa3-check: Aktiv", einen weißen Button "Deaktivieren" und zwei Dropdown-Menüs.

> Wenn sie inaktiv ist, fehlt das Tag und es gibt einen violetten Button :btn:Aktivieren:.

Reviewer: ```suggestion
Wenn sie inaktiv ist, fehlt das Tag und es gibt einen lilanen Button :btn:Aktivieren:.
```

Author: Ich finde das auch natürlich so, aber der Duden sagt, Adjektive wie "lila" zu flektieren ist umgangssprachlich. Daher: 
```suggestion
Wenn sie inaktiv ist, fehlt das Tag und es gibt einen lila Button :btn:Aktivieren:.
```

Vergewissern Sie sich, dass die Erweiterung aktiv ist.

> ![Seite 'Zahlungseinstellungen'. Der Reiter 'Zahlungsdienstleister' ist geöffnet und zeigt eine Liste mit den folgenden Einträgen: Banküberweisung, Wertgutschein, PayPal, SEPA-Lastschrift und Stripe. Die Option 'Wertgutschein' ist aktiv, alle anderen Einträge sind inaktiv. Neben allen Einträgen ist ein Button 'Einstellungen'.](../../assets/screens/payment-providers/payment-settings.png "Zahlungseinstellungen" )

Author: UI-Text, kleine Präzisierung
```suggestion
![Seite 'Zahlungseinstellungen'. Der Reiter 'Zahlungsmethoden' ist geöffnet und zeigt eine Liste mit den folgenden Einträgen: Banküberweisung, Wertgutschein, PayPal, SEPA-Lastschrift und Stripe. Die Option 'Wertgutschein' ist aktiv, alle anderen Einträge sind inaktiv. Neben jedem der Einträge ist ein Button 'Einstellungen'.](../../assets/screens/payment-providers/payment-settings.png "Zahlungseinstellungen" )
```

> ![Seite 'Zahlungseinstellungen'. Der Reiter 'Zahlungsdienstleister' ist geöffnet und zeigt eine Liste mit den folgenden Einträgen: Banküberweisung, Wertgutschein, PayPal, SEPA-Lastschrift und Stripe. Die Option 'Wertgutschein' ist aktiv, alle anderen Einträge sind inaktiv. Neben allen Einträgen ist ein Button 'Einstellungen'.](../../assets/screens/payment-providers/payment-settings.png "Zahlungseinstellungen" )

Reviewer: ```suggestion
![Seite 'Zahlungseinstellungen'. Der Reiter 'Zahlungsmethoden' ist geöffnet und zeigt eine Liste mit den folgenden Einträgen: Banküberweisung, Wertgutschein, Mollie, PayPal, SEPA-Lastschrift und Stripe. Die Option 'Wertgutschein' ist aktiv, alle anderen Einträge sind inaktiv. Neben jedem der Einträge ist ein Button 'Einstellungen'.](../../assets/screens/payment-providers/payment-settings.png "Zahlungseinstellungen" )
```

Auf dem Screenshot kommt zwischen „Wertgutschein“ und „PayPal“ erst noch „Mollie“. 
Fehlt das im englischen Text oder ist es bei der Übersetzung verloren gegangen??

Author: Mollie fehlt im englischen Text

Sie können direkt zu den Einstellungen für Banküberweisungen springen, indem Sie das Dropdown-Menü :btn-icon:fa3-gear:Einstellungen: und anschließend :btn:Zahlung > Banküberweisung: klicken.

Alternativ navigieren Sie zu :navpath:Ihre Veranstaltung →  :fa3-wrench: Einstellungen → Zahlung:.
Der Reiter :btn:Zahlungsdienstleister: auf dieser Seite zeigt die Liste aktiver Zahlungsdienstleister.
Die Liste sollte jetzt einen Eintrag für Banküberweisungen mit einem roten Tag ":fa3-remove: Deaktiviert" enthalten.

Die Erweiterung ist zwar aktiv, aber Sie haben Banküberweisungen für die Veranstaltung noch nicht als Zahlungsdienstleister eingerichtet.
Klicken Sie den Button :btn-icon:fa3-gear:Einstellungen: neben "Banküberweisung".
Dadurch gelangen Sie zu der Seite, auf der Sie Banküberweisungen einrichten.

> Wählen Sie den "Kontotyp".

Author: UI-Text
```suggestion
Wählen Sie die "Art der Bankverbindung".
```

Wenn Sie "SEPA-Bankkonto" wählen, müssen Sie den Namen von Kontoinhaber\*in, IBAN, BIC und den Namen der Bank angeben.
Wenn Sie "Anderes Bankkonto" wählen, müssen Sie in den Feldern bei "Angaben zum Bankkonto" die vollständigen Angaben der Bankverbindung eintragen.

Alle Einstellungen weiter unten auf der Seite sind optional.
Sehen Sie sich die Seite genau an und aktivieren Sie alle Einstellungen, die Sie für Ihre Veranstaltung für diesen Zahlungsdienstleister wünschen.

> Wenn alles wie gewünscht eingerichtet ist, blättern Sie zum Seitenanfang. Dort kreuzen Sie das Kontrollkästchen an, mit dem Sie bestätigen, dass Sie verstanden haben, wie Banküberweisungen in pretix funktionieren, und auch das Kontrollkästchen neben "Zahlungsmethode aktivieren".

Author: Eine Zeile pro Satz. Scrollen gefällt mir persönlich besser als blättern, aber wir besrpechen das intern. UI-Text.

```suggestion
Sobald Sie alles wie gewünscht eingerichtet haben, scrollen Sie zum Seitenanfang. 
Dort kreuzen Sie das Kontrollkästchen an, mit dem Sie bestätigen, dass Sie verstanden haben, wie Banküberweisungen in pretix funktionieren, und auch das Kontrollkästchen neben "Aktiviere Zahlungsmethode".
```

Reviewer: Ergbnis der internen Besprechung: "Scrollen", da "blättern" einen Wechsel zwischen Seiten suggeriert (statt Bewegung auf einer großen Seite).

> Den Kund\*innen stehen in Ihrem Shop nun Banküberweisungen als Zahlungsoption zur Verfügung.

Author: ```suggestion
Ihren Kund\*innen stehen in Ihrem Shop nun Banküberweisungen als Zahlungsoption zur Verfügung.
```

### Zahlungseingänge automatisch überwachen

> Für pretix Hosted arbeiten wir mit [GoCardless](https://gocardless.com) zusammen, um eine nahtlose Integration zu gewährleisten. Dadurch können Sie bei [mehreren Tausend Banken in über 30 Ländern](https://gocardless.com/bank-account-data/coverage/) Transaktionen automatisch importieren.

Author: Eine Zeile pro Satz. Außerdem brauchen wir die inhaltliche Beziehung zwischen "nahtlose Integration" und "automatischen Import"
```suggestion
Für pretix Hosted arbeiten wir mit [GoCardless](https://gocardless.com) zusammen, um eine nahtlose Integration des automatischen Imports von Transkationen zu gewährleisten. 
Diese Funktion ist mit [mehreren Tausend Banken in über 30 Ländern](https://gocardless.com/bank-account-data/coverage/) kompatibel.
```

> Für pretix Hosted arbeiten wir mit [GoCardless](https://gocardless.com) zusammen, um eine nahtlose Integration zu gewährleisten. Dadurch können Sie bei [mehreren Tausend Banken in über 30 Ländern](https://gocardless.com/bank-account-data/coverage/) Transaktionen automatisch importieren.

Reviewer: ```suggestion
Für pretix Hosted arbeiten wir mit [GoCardless](https://gocardless.com) zusammen, um eine nahtlose Integration zu gewährleisten. Dadurch können Sie von [mehreren Tausend Banken in über 30 Ländern](https://gocardless.com/bank-account-data/coverage/) Transaktionen automatisch importieren.
```

Vielleicht weniger schön, mein Gedanke ist: Die Daten wandern von der Bank ins pretix, nicht von pretix gen Bank.

Author: Ja. Möglicherweise könnten wir sogar noch präzisieren:
```suggestion
Für pretix Hosted arbeiten wir mit [GoCardless](https://gocardless.com) zusammen, um eine nahtlose Integration zu gewährleisten. Dadurch können Sie von [mehreren Tausend Banken in über 30 Ländern](https://gocardless.com/bank-account-data/coverage/) Transaktionen automatisch nach pretiximportieren.

```

> Alternativ haben Sie die Möglichkeit, regelmäßig digitale Kontoauszüge zu importieren, um pretix über Zahlungseingänge zu informieren.

Reviewer: > Alternativ haben Sie die Möglichkeit, regelmäßig digitale Kontoauszüge zu importieren, um pretix über Zahlungseingänge zu informieren.

Ließe sich durch „können Sie“ kürzen

Author: ```suggestion
Alternativ können Sie regelmäßig digitale Kontoauszüge importieren, um pretix über Zahlungseingänge zu informieren.
```

Dieser Abschnitt erläutert die Vorgehensweise bei beiden Optionen.

#### Option A: Transaktionen mit GoCardless automatisch importieren

<!-- md:hosted -->

Wenn Sie pretix Hosted nutzen, können Sie den automatischen Import von Transaktionen über die pretix-Integration mit GoCardless aktivieren.

> Diese Funktion gilt auf Veranstalterebene. Das bedeutet, dass die über dieses Verfahren importierten Bankdaten für alle Veranstaltungen des Veranstalters verfügbar sind.

Author: Eine Zeile pro Satz
```suggestion
Diese Funktion gilt auf Veranstalterebene. 
Das bedeutet, dass die über dieses Verfahren importierten Bankdaten für alle Veranstaltungen des Veranstalters verfügbar sind.
```

Navigieren Sie zu :navpath:Ihr Veranstalter → :fa3-bank: Banküberweisung → Automatischer Import:.

![Seite mit dem Titel 'Automatischer Import von Transaktionen'. Die Seite enthält Text mit den Unterüberschriften 'Warum Transaktionen automatisch importieren?', 'Nicht zugeordnete Zahlungen' und 'Sicherheit und Datenschutz', außerdem einen Button, um nicht zugeordnete Zahlungen zur Prüfung hochzuladen.](../../assets/screens/payment/automatic-transaction-import.png "Automatischer Import von Transaktionen" )

Wählen Sie Ihr Land, Ihre Bank und das Datum, ab dem Sie Transaktionen importieren möchten.

> Wenn Sie das Feld "Transaktionen importieren seit" leer lassen, importiert pretix so viele Transaktionen wie möglich.

Author: UI-Text
```suggestion
Wenn Sie das Feld "Transaktionen ab diesem Datum importieren" leer lassen, importiert pretix so viele Transaktionen wie möglich.
```

Das sind in der Regel alle Transaktionen der letzten 90 Tage.

Dies kann zu Problemen führen, falls Sie Bankdaten bisher mit einem anderen Verfahren in dasselbe Veranstalterkonto importiert haben.
Möglicherweise sind die Daten unterschiedlich formatiert, wodurch die Software Transaktionen eventuell nicht als identisch erkennt.
Wenn das bei Ihnen der Fall ist, wählen Sie den ersten Tag, für den Sie in dieses Veranstalterkonto noch keine Bankdaten importiert haben.
Lassen Sie keine Lücke zwischen alten und neuen Importen, etwa wenn Sie für einen Tag bereits vor dem Buchungsschluss Daten importiert haben.

Falls Sie bisher noch kein anderes Verfahren für den Import von Bankdaten verwendet haben, können Sie dieses Feld leer lassen.

Klicken Sie den Button :btn-icon:fa3-sign-in: Mit Bank verbinden:.
Damit gelangen Sie zu einer Webseite bei ob.gocardless.com, die Sie auffordert, der Verarbeitung Ihrer Daten durch GoCardless zuzustimmen und die Anmeldedaten für Ihre Bank einzugeben.

> Befolgen Sie zum Abschluss der Autorisierung die Anweisungen der Website.

Author: Handlung und Ziel sauber voneinander trennen und nicht miteinander verschränken
```suggestion
Befolgen Sie die Anweisungen auf der Website, um die Autorisierung abzuschließen.
```

Die Website leitet Sie dann wieder zurück zum Backend von pretix.

Wenn Ihr GoCardless-Konto Angaben für mehrere Konten bei derselben Bank enthält, müssen Sie das Bankkonto wählen, aus dem Sie Daten importieren möchten.

> Wählen Sie ein Startdatum zum Import der Transaktionen und bestätigen Sie den Vorgang, indem Sie den Button :btn:Mit Bank verbinden: klicken.

Author: Fehlendes Icon im Originaltext
```suggestion
Wählen Sie ein Startdatum zum Import der Transaktionen und bestätigen Sie den Vorgang, indem Sie den Button :btn-icon:fa3-sign-in:Mit Bank verbinden: klicken.
```

Wenn Sie jetzt zurück zur Seite "Automatischer Import" navigieren, werden dort die Bankkonten angezeigt, die Sie auf der vorigen Seite als verbunden gewählt haben.

> Wenn der automatische Import aktiv ist, sollten Sie gelegentlich nachsehen, ob es ungeklärte Transaktionen gibt.

Author: ```suggestion
Wenn der automatische Import aktiv ist, sollten Sie gelegentlich nachsehen, ob es unzugeordnete Überweisungen gibt.
```

> Die häufigste Ursache für ungeklärte Transaktionen sind falsch geschriebene oder fehlende Bestellnummern im Verwendungszweck.

Author: ```suggestion
Die häufigste Ursache für unzugeordnete Überweisungen sind falsch geschriebene oder fehlende Bestellnummern im Verwendungszweck.
```

> Falls das Bankkonto auch für Transaktionen genutzt wird, die nichts mit Verkäufen über pretix zu tun haben, entstehen auch dadurch ungeklärte Transaktionen.

Author: ```suggestion
Falls das Bankkonto auch für Transaktionen genutzt wird, die nichts mit Verkäufen über pretix zu tun haben, entstehen auch dadurch unzugeordnete Überweisungen.
```

pretix gibt Ihnen die Möglichkeit, diese Transaktionen manuell zu verarbeiten.

> Der folgende Abschnitt erklärt, wie Sie [ungeklärte Transaktionen manuell verarbeiten](bank-transfer.md#handling-unresolved-transactions).

Author: ```suggestion
Der folgende Abschnitt erklärt, wie Sie [unzugeordnete Überweisungen manuell verarbeiten](#unzugeordnete-überweisungen-manuell-bearbeiten).
```

#### Option B: Bankdaten importieren

Besorgen Sie sich einen Export der Transaktionsdaten Ihres Bankkontos.
Dieser Export muss eine Datei im Format CSV oder MT940 sein.
Sie muss folgende Angaben enthalten:

 - Datum
 - Betrag
 - Verwendungszweck

>  - Zahler\*in

Author: UI-Text
```suggestion
 - Zahlender
```

 - IBAN
 - BIC

> Geben Sie bei "Zahler\*in" den Namen der Person an, von der die Zahlung stammt.

Author: Die Datei sollte diese Daten bereits enthalten. Wenn man sie händisch bearbeiten muss, ist was schief gelaufen 
```suggestion
Die Zeile "Zahlender" sollte den namen der Person enthalten, von der die Zahlung stammt. 
```

IBAN und BIC sind IBAN und BIC des Bankkontos, von dem die Zahlung stammt.
IBAN und BIC sind optional, aber wenn Sie sie angeben, können Sie mehr Funktionen in pretix nutzen, etwa die automatische Erstellung von Erstattungsdateien.

Sie haben zwei Optionen: Bankdaten für alle Veranstaltungen auf der Veranstalterebene importieren oder Bankdaten für eine Einzelveranstaltung importieren.
Wir empfehlen die erste Option, außer Sie haben für jede Ihrer Veranstaltungen ein eigenes Bankkonto.

Wenn Sie Daten für **alle Veranstaltungen** eines Veranstalters importieren möchten, navigieren Sie zu :navpath:Ihr Veranstalter → :fa3-bank: Banküberweisung:.
Sie gelangen zu einer Seite mit dem Titel "Bankdaten importieren".

Wenn Sie die Daten für eine **Einzelveranstaltung** importieren möchten, navigieren Sie zu :navpath:Ihre Veranstaltung → :fa3-bank: Banküberweisung:.
Auch hier gelangen Sie zu einer Seite mit dem Titel "Bankdaten importieren".

> Die beiden Seiten sind sehr ähnlich, aber eine gilt für die Ebene der Veranstaltung, die andere für die des Veranstalters.

Reviewer: ```suggestion
Die beiden Seiten sind sehr ähnlich, aber eine befindet sich auf Ebene der Veranstaltung, die andere auf Ebene des Veranstalters.
```

Üblicherweise "Veranstaltungsebene" und "Veranstatlerebene"; im Direktvergleich vielleicht so klarer?

Author: Eigentlich geht es hier gar nicht um die verschiedenen Ebenen im UI. Entscheidend ist ja, ob man Bankdaten für die einzelne Veranstaltung oder für das gesamte Veranstalterkonto importieren möchte. 

```suggestion
Die beiden Seiten sind sehr ähnlich, aber eine gilt für die einzelne Veranstaltung, die andere für das ganze Veranstalterkonto.
```

Grundsätzlich wäre ich für die Schreibweisen "Veranstaltungs-Ebene" und "Veranstalter-Ebene" da die drei unterschiedlichen Buchstaben mitten im Wort sonst schnell mal überlesen werden

Unabhängig von der gewählten Option ist die Vorgehensweise ab hier dieselbe.

Klicken Sie den Button :btn:Durchsuchen…: und wählen Sie die Exportdatei, die Sie hochladen möchten.

> Klicken Sie den Button :btn:Upload starten:.

Author: ```suggestion
Klicken Sie den Button :btn:Hochladen:.
```

pretix fordert Sie jetzt auf anzugeben, welche Spalte Ihrer Datei welche Daten enthält.
Der folgende Screenshot zeigt am Beispiel einer kleinen CSV-Datei, wie eine solche Zuordnung aussehen kann.

![Seite 'Bankdaten importieren' mit einem Dialogfeld, das auffordert, Spalten einer CSV-Datei den Datenkategorien Datum, Betrag, Verwendungszweck, Zahler\*in, IBAN und BIC zuzuordnen. Radiobuttons und Kontrollkästchen ordnen die einzelnen Datenkategorien der Spalte der CSV-Datei zu, die die entsprechenden Angaben enthält. ](../../assets/screens/payment-providers/import-bank-data.png "Bankdaten importieren" )

> Klicken Sie :btn:Weiter:.

Author: ```suggestion
Klicken Sie :btn:Fortfahren:.
```

Während Ihre Daten verarbeitet werden, zeigt pretix einen Ladebildschirm und anschließend einen Überblick, wie viele Bestellungen als bezahlt oder ungültig erkannt oder ignoriert wurden.
Das System ignoriert Transaktionen, die Sie bereits zuvor mit demselben Verfahren importiert haben.

Bei diesem Verfahren kann es gelegentlich ungeklärte Transaktionen geben.
Die häufigste Ursache dafür sind falsch geschriebene oder fehlende Bestellnummern im Verwendungszweck.
Falls das Bankkonto auch für Transaktionen genutzt wird, die nichts mit Verkäufen über pretix zu tun haben, entstehen auch dadurch ungeklärte Transaktionen.

> In diesem Bildschirm können Sie eingreifen und Fehler manuell korrigieren.

Author: ```suggestion
In dieser Ansicht können Sie eingreifen und Fehler manuell korrigieren.
```

Eine ausführlichere Anleitung, wie Sie [ungeklärte Transaktionen manuell verarbeiten](bank-transfer.md#handling-unresolved-transactions), finden Sie im Abschnitt weiter unten.

### Transaktionen manuell freigeben

![Eine Seite mit dem Titel 'Bestellungen' zeigt eine Liste mit einer Bestellung. Deren Status ist 'ausstehend', 0,00 € von 250,00 € bezahlt.](../../assets/screens/payment/orders.png "Bestellungen" )

> Ergänzend zur [automatischen Option A](bank-transfer.md#option-a-automatic-transaction-import-using-gocardless) und zur [halbautomatischen Option B](bank-transfer.md#option-b-importing-bank-data), die beide oben beschrieben sind, haben Sie in pretix auch die Möglichkeit, Transaktionen manuell freizugeben.

Author: ```suggestion
Ergänzend zur [automatischen Option A](#option-a-transaktionen-mitgocardless-automatisch-importieren) und zur [halbautomatischen Option B](bank-transfer.md#option-b-bankdaten-importieren), die beide oben beschrieben sind, haben Sie in pretix auch die Möglichkeit, Transaktionen manuell freizugeben.
```

Dazu navigieren Sie zu :navpath:Ihre Veranstaltung → :fa3-shopping-cart: Bestellungen:.
Diese Seite zeigt alle Bestellungen an, die Kund\*innen in Ihrem Shop getätigt haben.

> Wenn ein Kund\*in eine Bestellung getätigt hat und Sie die Zahlung dafür noch nicht erhalten haben, zeigt die Seite diese Bestellung mit dem gelben Status-Tag ":fa3-money: Ausstehend" an.

Author: ```suggestion
Wenn ein\*e Kund\*in eine Bestellung getätigt hat und Sie die Zahlung dafür noch nicht erhalten haben, zeigt die Seite diese Bestellung mit dem gelben Status-Tag ":fa3-money: Ausstehend" an.
```

Für Bestellungen, die per Banküberweisung bezahlt werden sollen, ist dies zu erwarten.

Klicken Sie die Bestellnummer einer der ausstehenden Bestellungen.

> Dadurch gelangen Sie zur Seite "Bestellungsdetails" für diese Bestellung.

Author: Deutsche Übersetzung des Seitentitels "Order details" ist "Bestellung". Aaarrrgh. 
```suggestion
Dadurch gelangen Sie zu einer Seite mit Details über diese Bestellung.
```

Prüfen Sie die Transaktionsdaten Ihres Bankkontos.
Wenn die Transaktionsdaten Ihres Bankkontos einen Eintrag enthalten, der zu der fraglichen Bestellung passt, klicken Sie oben auf der Seite mit den Bestellungsdetails den Button :btn-icon:fa3-check:Als bezahlt markieren:.

> ![Seite mit dem Titel 'Bestellung als bezahlt markieren'. Sie zeigt Optionen zum Ändern des Zahlungsbetrags von 250,00 € und des Zahlungsdatums sowie die Option, den Kund\*in per E-Mail zu informieren.](../../assets/screens/payment/order-mark-as-paid.png "Bestellungen" )

Author: ```suggestion
![Seite mit dem Titel 'Als bezahlt markieren'. Sie zeigt Optionen zum Ändern des Zahlungsbetrags von 250,00 € und des Zahlungsdatums sowie die Option, den\*die Kund\*in per E-Mail zu informieren.](../../assets/screens/payment/order-mark-as-paid.png "Bestellungen" )
```

!!! Warnung

>     Vergewissern Sie sich, bevor Sie die Bestellung als bezahlt markieren, dass Bestellung, Preis und Datum korrekt sind.

Author: Eingebettete Nebensätze vermeiden
```suggestion
    Bevor Sie die Bestellung als bezahlt markieren, sollten Sie sich vergewissern, dass Bestellung, Preis und Datum korrekt sind.
```

    Nachdem pretix eine Zahlung erfasst hat, ist es **nicht** mehr möglich, sie zu löschen.

Passen Sie Betrag und Datum der Zahlung bei Bedarf an.
Standardmäßig informiert pretix den Kund\*in per E-Mail darüber, dass die Bestellung als bezahlt markiert wurde.

> Sie können das verhindern, indem Sie den Haken aus dem Kontrollkästchen neben "Kund\*in per E-Mail informieren" entfernen.

Author: ```suggestion
Sie können das verhindern, indem Sie das Kontrollkästchen neben "Kund\*in per E-Mail benachrichtigen" deaktivieren.
```

Bestätigen Sie den Zahlungseingang, indem Sie den Button :btn:Zahlung erstellen: klicken.

> Dadurch gelangen Sie zurück zur Seite "Bestellungsdetails", die oben jetzt die Nachricht "Die Zahlung wurde erstellt" anzeigt und oben rechts ein grünes Tag mit dem Text ":fa3-check: Bezahlt".

Author: ```suggestion
Dadurch gelangen Sie zurück zur Seite "Bestellungsdetails", die oben jetzt die Nachricht "Die Zahlung wurde erfolgreich verbucht" anzeigt und oben rechts ein grünes Tag mit dem Text ":fa3-check: Bezahlt".
```

> Wiederholen Sie diese Vorgehensweise für jede auf der Seite "Bestellungen" aufgeführte Bestellung.

Author: Man kann ein Vorgehen wiederholen, aber eine Vorgehensweise? Klingt komisch
```suggestion
Wiederholen Sie dieses Vorgehen für jede auf der Seite "Bestellungen" aufgeführte Bestellung.
```

> ### Ungeklärte Transaktionen manuell verarbeiten

Author: ```suggestion
### Unzugeordnete Überweisungen manuell bearbeiten
```

Sie können ungeklärte Transaktionen auf Veranstalter- oder auf Veranstaltungsebene verarbeiten.

> Wenn Sie den automatischen Import über **GoCardless** nutzen oder Bankdaten auf **Veranstalterebene** importiert haben, navigieren Sie zu :navpath:Ihr Veranstalter → :fa3-bank: Banküberweisung:. Sie gelangen zu einer Seite mit dem Titel "Bankdaten importieren".

Author: ```suggestion
Wenn Sie den automatischen Import über **GoCardless** nutzen oder Bankdaten auf **Veranstalterebene** importiert haben, navigieren Sie zu :navpath:Ihr Veranstalter → :fa3-bank: Banküberweisung:. 
Sie gelangen zu einer Seite mit dem Titel "Bankdaten importieren".
```

> Wenn Sie Bankdaten auf **Veranstaltungsebene** importiert haben, navigieren Sie zu :navpath:Ihre Veranstaltung → :fa3-bank: Banküberweisung:. Sie gelangen zu einer Seite mit dem Titel "Bankdaten importieren".

Author: ```suggestion
Wenn Sie Bankdaten auf **Veranstaltungsebene** importiert haben, navigieren Sie zu :navpath:Ihre Veranstaltung → :fa3-bank: Banküberweisung:. 
Sie gelangen zu einer Seite mit dem Titel "Bankdaten importieren".
```

Ab hier ist die Vorgehensweise dieselbe.

> Wenn es ungeklärte Transaktionen gibt, zeigt diese Seite sie in der Liste "Ungeklärte Transaktionen" an.

Author: ```suggestion
Wenn es unzugeordnete Überweisungen gibt, zeigt diese Seite sie in der Liste "Unzugeordnete Überweisungen" an.
```

Sie haben für jede Transaktion in der Liste zwei Optionen:
Sie können pretix anweisen, die Transaktion zu ignorieren, indem Sie den Butten :btn-icon:fa3-trash:: klicken, oder Sie können nach der passenden Bestellung suchen und der Transaktion die Bestellnummer zuordnen.

Dazu verwenden Sie das Feld "Bestellnummer" der ungeklärten Transaktion.
Sie können einen Teil der Bestellnummer, des Namens des bzw. der Käufer\*in, den Namen eines bzw. einer Teilnehmer\*in oder der E-Mail-Adresse eingeben, über die die Bestellung getätigt wurde.
Wählen Sie die passende Bestellung aus den angezeigten Suchergebnissen und klicken Sie zur Bestätigung den Button :btn-icon:fa3-check::.

> Wiederholen Sie diese Vorgehensweise für alle ungeklärten Bestellungen.

Author: ```suggestion
Wiederholen Sie dieses Vorgehen für alle unzugeordneten Überweisungen.
```
